amundsen_application/static/js/components/common/ResourceList/ResourceListHeader/index.tsx

```diff
@@ -5,21 +5,40 @@ import * as React from 'react';
 
 import './styles.scss';
 
+import { ResourceType } from 'interfaces';
 import {
   RESOURCE_HEADER_TITLE,
   SOURCE_HEADER_TITLE,
   BADGES_HEADER_TITLE,
+  LAST_UPDATED_HEADER_TITLE,
 } from './constants';
 
-const ResourceListHeader: React.FC = () => {
+export interface ResourceListHeaderProps {
+  resourceTypes: ResourceType[];
+}
+
+const contentHeaderTitle = (type: ResourceType): string => {
+  switch (type) {
+    case ResourceType.dashboard:
+      return LAST_UPDATED_HEADER_TITLE;
+
+    default:
+      return BADGES_HEADER_TITLE;
+  }
+};
```

Comment:
Nit: I wouldn't use a switch here, as a simple if would be enough.

Reply:
My thinking is that it would make it clearer where to add code if folks added new resource types in the future -- wdyt?

Reply:
Here I default to YAGNI, if that's the case, we could introduce the switch. (Although I would always prefer a mapped object instead)

Reply:
yeah that's fair, updated